aws/ec2/deploy/home/ec2-user/snappydata/resolve-hostname.sh

```diff
@@ -14,28 +14,28 @@
 #     exit 0
 # fi
 
-PRIVATE_IP=`wget -q -O - http://169.254.169.254/latest/meta-data/local-ipv4`
+#PRIVATE_IP=`wget -q -O - http://169.254.169.254/latest/meta-data/local-ipv4`
```

Comment:
I think if we are not doing anything here, better to comment it's invocation from aws-setup.sh instead of commenting all the code here.